app/helpers/data_tables_helper.rb

```diff
@@ -3,7 +3,8 @@ def datatables_source
     data_tables_path(
       query: params[:query],
       fetch: params[:fetch],
-      model: model_param
+      model: model_param,
+      format: :json
```

Comment:
You may want to remove the new param from cache key in the index operation - though I guess this is not needed, since it does not change anyway.